examples/algorithms/plot_parallel_betweenness.py

```diff
@@ -10,8 +10,12 @@
 the contribution of those nodes to the betweenness centrality of the whole
 network. Here we divide the network in chunks of nodes and we compute their
 contribution to the betweenness centrality of the whole network.
-"""
 
+Note: The example output below shows that the non-parallel implementation is
+faster. This is a limitation of our CI/CD pipeline running on a single core.
+
+Depending on your setup, you may observe a 4x speed improvement.
```

Comment:
```you may observe a 4x speed improvement.``` 
is both specific (x4) and qualified (you may observe).  Which feels strange.
What do you think about:   
```Depending on your setup, you will likely observe a speedup. We seem to get a 4x improvement.```

Really, I'm fine either way -- or another way.  Thoughts?

Reply:
Agreed - I don't think we should get too specific. Ultimately the performance will depend on the hardware (and even more complicating factors like resource limitations for people running in containerized environments), so I'd keep the wording as generic as possible --- maybe @dschult's second suggestion (potentially even dropping the "We seem to get..." part).

Reply:
Thanks! Apologies for delay in responding, dealing with family emergency. Might consider adding `concurrent.futures` example in future PR -- looking forward to collaborating further